src/components/UI/molecules/CardDivider/CardDivider.jsx

```diff
@@ -0,0 +1,80 @@
+/* eslint-disable no-use-before-define */
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+import FlexContainer from '<atoms>/layouts/FlexContainer/FlexContainer';
+import Divider from '<atoms>/Divider/Divider';
+import Text from '<atoms>/Text/Text';
+import spacing from '<variables>/spacing';
+import { fontSizes } from '<variables>/fonts';
+
+/**
+ * @description - CardDivider Component
+ *
+ * @prop {string} text - divider text
+ * @prop {string} width - maxWidth
+ * @prop {string} margin - margin
+ * @prop {string} fontSize - font Size
+ * @prop {string} display - container display
+ *
+ * @return {component} CardDivider
+ */
+const CardDivider = ({
+  text,
+  width,
+  margin,
+  fontSize,
+  display,
+}) => {
+  return (
+    <CardDivider.Container>
+      <FlexContainer
+        display={display}
+        margin={margin}
+      >
+        { renderDivider() }
```

Comment:
I don't think there should be space around the braces, please configure this rule in eslint